menpofit/io.py

```diff
@@ -181,9 +253,25 @@ def fit_from_shape(self, image, initial_shape, **kwargs):
         final_kwargs = self._fit_from_shape_kwargs.copy()
         # If the user provided kwargs at runtime, they take precedence.
         final_kwargs.update(kwargs)
-        # call the wrapped fitter with the updated kwargs
-        return self.wrapped_fitter.fit_from_shape(image, initial_shape,
-                                                  **final_kwargs)
+        # check if pre-processing function exists
+        if self._image_preprocess is None:
+            # call the wrapped fitter with the updated kwargs
+            result = self.wrapped_fitter.fit_from_bb(image, initial_shape,
+                                                     **final_kwargs)
+        else:
+            # pre-process the image
+            proc_image, trans = self._image_preprocess(image, initial_shape)
+            # call the wrapped fitter with the updated kwargs
+            result = self.wrapped_fitter.fit_from_bb(
```

Comment:
Thanks mate! Fixed in 02b3b88448cb01765eba4e11720a9a4c6e19d6a9 (hotfix on master)